### PR TITLE
Search page improvements

### PR DESCRIPTION
**Pull request by osamasayed**
### Summary
This PR adds support to language and translation inside search page and also handles showing the results when there are some navigation results but no verses in the response.

### Screenshots

| Before | After |
| ------ | ------ |
|Screen Shot 2021-09-06 at 19 33 55|Screen Shot 2021-09-06 at 19 35 44|
|Screen Shot 2021-09-06 at 19 38 57|Screen Shot 2021-09-06 at 19 39 02|

### CODE REVIEW

File: src/components/Navbar/SearchDrawer/SearchDrawer.tsx [hunk 1]
```diff
@@ -121,11 +121,12 @@ const SearchDrawer: React.FC = () => {
     });
   }, [closeSearchDrawer, router.events, isOpen]);
 
+  const noResponseResults =
```
Comment (tifa2UP) [inline src/components/Navbar/SearchDrawer/SearchDrawer.tsx:124]: what's a better name for this?

**Comment by muhajirdev** [comment]
tested the functionality

File: src/components/Navbar/SearchDrawer/SearchDrawer.tsx (hunk 1 above)
Comment (osamasayed) [reply src/components/Navbar/SearchDrawer/SearchDrawer.tsx:124]: How about `isEmptyResponse`?

**Review by tifa2UP — APPROVED** [review]

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**/search**](https://quran-com-frontend-next-ehssx7x6m-qurancom.vercel.app/search) |
|:-:|
|<a href="https://quran-com-frontend-next-ehssx7x6m-qurancom.vercel.app/search">Screenshot of /search</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/A0KStU8UJRWbHtpkeL1uZz8z0rLqnZupMyptsmNRGRgi3YsaANN36_6VT5-McflfGHqWxNN45EGPeoSKJn4cfA.png">(view full size)</a> |







Commit 5655936710d3e3fedf907c12d4749a973cb82dc8 (https://quran-com-frontend-next-ehssx7x6m-qurancom.vercel.app).

**Pull request merged**